package.json

```diff
@@ -17,6 +17,7 @@
     "cookie-parser": "^1.4.3",
     "cors": "^2.8.4",
     "express": "^4.16.4",
+    "googleapis": "^45.0.0",
```

Comment:
We don't end up using this correct?

Reply:
Yep, you're right. Removed it from this file and `package-lock.json` and scheduling still works.